server/src/domain/asset/response-dto/asset-response.dto.ts

```diff
@@ -42,9 +42,13 @@ export class AssetResponseDto extends SanitizedAssetResponseDto {
   people?: PersonResponseDto[];
   /**base64 encoded sha1 hash */
   checksum!: string;
+  stackParentId?: string | null;
+  stack?: AssetResponseDto[];
+  @ApiProperty({ type: 'integer' })
+  stackCount!: number;
 }
 
-export function mapAsset(entity: AssetEntity, stripMetadata = false): AssetResponseDto {
+export function mapAsset(entity: AssetEntity, stripMetadata = false, withStack = false): AssetResponseDto {
```

Comment:
I think we should change this to be `{ stripMetadata: boolean, withStack: boolean }` instead of two boolean params, as it is hard to know what each one does in other files that call this function.

Reply:
Updated.